module/settings.mjs

```diff
@@ -440,6 +440,21 @@ export function registerSystemSettings() {
     default: [],
     config: true
   });
+
+  // Auto roll NPC HP on drop
+  game.settings.register("dnd5e", "autoRollNPCHP", {
+    name: "SETTINGS.DND5E.AUTOROLLNPCHP.Name",
+    hint: "SETTINGS.DND5E.AUTOROLLNPCHP.Hint",
+    scope: "world",
+    config: true,
+    default: "no",
+    type: String,
+    choices: {
+      no: "SETTINGS.DND5E.AUTOROLLNPCHP.No",
+      silent: "SETTINGS.DND5E.AUTOROLLNPCHP.Silent",
+      yes: "SETTINGS.DND5E.AUTOROLLNPCHP.Yes"
+    }
```

Comment:
Could (should?) use a data field here.

Reply:
Just based mine on the existing single-select dropdowns. Happy to change if necessary though - what's the benefit of doing it that way? (Might also make sense as a separate maintenance PR that changes _all_ relevant settings to use data fields.)

Reply:
DataFields have two big upsides
- Functions for initial value & `choices` (more useful when you're doing things like picking a document ID)
- Supports a labeled blank option.